consensus/spos/commonSubround/subroundStartRound.go

```diff
@@ -77,7 +77,9 @@ func checkNewSubroundStartRoundParams(
 
 // SetAppStatusHandler method set appStatusHandler
 func (sr *SubroundStartRound) SetAppStatusHandler(handler core.AppStatusHandler) {
```

Comment:
The pattern already used in app status handlers impl is that this func should return error if the provided handler is nil. Let's keep this pattern.

Reply:
SetAppStatusHandler return error now